qa/acceptance/spec/centos/lib/install_spec.rb

```diff
@@ -1,13 +1,13 @@
 # encoding: utf-8
-require_relative '../spec_helper'
-require          'logstash/version'
+require_relative "../spec_helper"
+require "logstash/version"
 
 describe "artifacts", :platform => :centos do
 
   shared_examples "installable" do |host, name|
 
     before(:each) do
-      install("/home/vagrant/logstash-latest-SNAPSHOT.rpm", host)
+      install("/logstash-build/logstash-#{LOGSTASH_VERSION}.noarch.rpm", host)
```

Comment:
This revert the previous commit. https://github.com/elastic/logstash/commit/795d5d0619b57431179afc50fdb18172af505f4b#diff-2aa36be24fd3a114b68db38ebed4a070R10